.github/workflows/pull-request-requirements.yml

```diff
@@ -0,0 +1,26 @@
+name: Check PR Requirements
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, labeled, unlabeled]
+
+jobs:
+  check-changelog:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip changelog') }}
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Install dependencies
+        run: npm install --prefix ./.github/actions
```

Comment:
Nit: if we're using Yarn and adding a Yarn lock file for the action, let's also use Yarn instead of NPM for the install for consistency.